### PR TITLE
Add support for froala editor

### PR DESCRIPTION
**Pull request by codynguyen**
This PR adds support for the wonderful Froala editor http://www.froala.com/. I'm using Froala in one of my projects and I have to say it's the best WYSIWYG editor out there.

### How to use:

1. Add Froala's assets gem. **Note: ** For now we'll need to use my fork, the assets are precompiled (which is needed for rails_admin). I have submitted a pull request and when it's merged into their official repo, we can remove the `github` in the line below.

  ```
    gem "wysiwyg-rails", github: "codynguyen/wysiwyg-rails"
  ```

2. Enabling froala editor for your field is easy

  ```
  edit do
    field :content, :froala do
      config_options Settings.default_froala_config
    end
  end

  # Optionally providing froala options (see https://editor.froala.com/options)
  edit do
    field :content, :froala do
      config_options {
        inlineMode: false,
        paragraphy: false
      }
    end
  end
```

### CODE REVIEW

File: lib/rails_admin/config/fields/types/text.rb [hunk 1]
```diff
@@ -26,6 +26,12 @@ class Text < RailsAdmin::Config::Fields::Base
             end
           end
 
+          [:froala, :froala_config_options, :froala_css_location, :froala_js_location].each do |key|
+            register_deprecated_instance_option key do
+              fail("The 'field(:foo){ froala true }' style DSL is deprecated. Please use 'field :foo, :froala' instead.")
+            end
+          end
```
Comment (mshibuya) [inline lib/rails_admin/config/fields/types/text.rb]: This part is compatibility purpose for older RailsAdmin configuration(there was [a DSL change](https://github.com/sferik/rails_admin/commit/967efc6f875c5f2075c697e22e894cad2fbc99b1) in 0.5.0), so not required for a newly introduced feature.

**Comment by codynguyen** [comment]
@mshibuya Thanks for the note. The unnecessary block has been removed :)

**Comment by mshibuya** [comment]
Thanks for contribution!
It would be nice if you could add instruction to Wiki :smiley:

**Comment by codynguyen** [comment]
@mshibuya instruction added here: https://github.com/sferik/rails_admin/wiki/Froala :)

**Comment by TangMonk** [comment]
language seems not works

**Pull request merged**